### DISCUSSION TITLE
Argocd gitlab "error testing repository connectivity: authorization failed"

### DISCUSSION THREAD

**Discussion by vc019** [argoproj/argo-cd#19589] [General]
Hi All,

I am facing a peculiar issue and not sure if it is a bug or not. 

Description:
When I try to register a repository from argocd-ui. The registration fails, and there is an error in the repo server log
"error testing connectivity: authorization failed". 
 
I have tried cloning the repo on the cluster nodes and it clones without an issue. But only inside argocd does it give an authorization error.

I am using EKS 1.29, argocd 2.10.7 and gitlab v17. Any pointers would be greatly appreciated.

Many Thanks

V

**Comment by Paulino02** [comment]
Did you manage to solve this problem? I have the same problem, here.
